docs/how_to/run_tests.rst

```diff
@@ -0,0 +1,26 @@
+#############
+Run the tests
+#############
+
+
+Backend Tests
+=============
+
+You can run the tests by following:
+
+* ``virtualenv env`` to create the virtual environment
+* ``source env/bin/activate`` to activate it
+* ``pip install -e test_requirements/django_17.txt`` to install the
+  requirements
```

Comment:
add one more step here:
* navigate to aldryn_events directory and run `python setup.py install` this would install aldryn_events into virtualenv and all it's dependencies.